kong/plugins/session/schema.lua

```diff
@@ -94,7 +94,7 @@ return {
           { request_headers = headers },
           { logout_methods = logout_methods },
           { logout_query_arg = {  type = "string",  default = "session_logout" } },
-          { logout_post_arg = { type = "string", default = "session_logout" } },
```

Comment:
my concern would be, as you pointed out, those who are relying on the removed default. Seems like manager might be one of them: https://github.com/Kong/kong-admin/blob/4dfc92fe311a21d6e24142e2d7807c71f3b768e7/src/stores/auth.ts#L73 (?)

Reply:
Manager should force the setting to whatever they need for this logout parameter, and not rely on user configuration on it. Manager owns the logout and they know how it is to be used. We need to check this with Manager code.

The line you pointed out uses `DELETE` and parameter is added to query, so this change should not affect at least that line of code. The `logout_query_arg` still has the default.

Reply:
Here is a link to EE code (not publicly visible):
https://github.com/Kong/kong-ee/blob/master/kong/api/routes/kong.lua#L398-L408

That `local session_conf`  should contain all the settings that Manager needs, e.g. it should force the `logout_post_arg=xxx` if it really needs that (despite what user might have configured - it should be a setting fully controlled by Manager code itself).

Reply:
> The line you pointed out uses DELETE and parameter is added to query, so this change should not affect at least that line of code
you are right. I made a quick search and noticed the same value, in that case applied to the query arg. Still not ideal that it appears to be relying on the default value of `logout_query_arg`.

> it should be a setting fully controlled by Manager code itself, not the user configuring manager
agree